lib/CLI/Types.hs

```diff
@@ -149,14 +146,14 @@ data SetFieldOptions = SetFieldOptions
   , sfoVisibility :: Maybe FieldVisibility
   }
   deriving stock (Show, Generic)
-  deriving anyclass (FromJSON, ToJSON)
+  deriving anyclass (ToJSON)
 
 data DeleteFieldOptions = DeleteFieldOptions
   { dfoQPath :: QualifiedPath EntryPath
   , dfoFieldName :: FieldName
   }
   deriving stock (Show, Generic)
-  deriving anyclass (FromJSON, ToJSON)
+  deriving anyclass (ToJSON)
```

Comment:
Looks like most of these `ToJSON` instances aren't needed either, might as well delete them

Reply:
Done